### DISCUSSION TITLE
Unable to connect to phone using Tailscale

### DISCUSSION THREAD

**Discussion by 0xMRTT** [andyholmes/valent#697] [Help]
### Current Behavior

I'm using Tailscale on my phone and on my computer, it's a mesh network so there's an IP for the phone and one for the computer. I tried to add the IP of the computer in the mobile app but it still doesn't detect Valent. Tailscale detect that Valent is running but the mobile app doesn't detect Valent. If you need more informations tell me!

### Expected Behavior

I except the mobile app to detect valent and valent to detect the mobile app.

### Desktop

GNOME Shell

### Operating System

NixOS

### Installed from

Distribution Package

### Version

1.0.0.alpha

### Devices

KDE Connect (Android)

**Comment by andyholmes** [comment]
Sorry, I don't really know anything about how Tailscale works, but it sounds interesting and a good fit for KDE Connect!

One way or the other, KDE Connect uses UDP for discovery (even with the new DNS-SD support). Can you confirm that UDP packets are not blocked for ports 1716-1764 and/or 5353?

**Comment by 0xMRTT** [comment]
Any ideas how can i check it ?

**Comment by andyholmes** [comment]
No idea :grimacing: I'm not sure how this technology works, maybe Wireshark?

**Comment by 0xMRTT** [comment]
Will try

**Comment by andyholmes** [comment]
Moving this to discussions, since it doesn't appear to be an issue with Valent's expected functionality.

If there's something special Valent *should* do, we can open a specific issue for that.

**Comment by kevinnlsamuel** [comment]
I have used it with Tailscale and it worked fine. It's likely that your firewall needs some tweaking @0xMRTT . what firewall does your desktop use?

**Reply by isshin1** [reply]
> I have used it with Tailscale and it worked fine. It's likely that your firewall needs some tweaking @0xMRTT . what firewall does your desktop use?

it does work as long as both device are connected to same network. 

Using android on mobile network, I am able to ping it from my pc, using its tailscale ip, but device disappears from kdeconnect.

are you able to connect from different networks ?

```
--                         ------      ----
1714:1764/udp              ALLOW       Anywhere                  
1714:1764/tcp              ALLOW       Anywhere                  
Anywhere on tailscale0     ALLOW       Anywhere                  
22                         ALLOW       Anywhere                  
SSH                        LIMIT       Anywhere                  
1714:1764/udp (v6)         ALLOW       Anywhere (v6)             
1714:1764/tcp (v6)         ALLOW       Anywhere (v6)             
Anywhere (v6) on tailscale0 ALLOW       Anywhere (v6)             
22 (v6)                    ALLOW       Anywhere (v6)             
SSH (v6)                   LIMIT       Anywhere (v6)        
```

my ufw rules 


edit: I entered pc tailscale ip manually in kdeconnect and it worked
